vms/platformvm/config/execution_config_test.go

```diff
@@ -12,6 +16,30 @@ import (
 	"github.com/ava-labs/avalanchego/vms/platformvm/network"
 )
 
+// Errors if all values in a struct are not initialized
+func VerifyInitializedStruct(s interface{}) error {
```

Comment:
(optional) Since this is in a test, you can accept `testing.TB` as a first parameter and just call `tb.Fatalf()` instead of returning errors. If you call `tb.Helper()` at the beginning of the function then error reports will have the line of the call site not of the `Fatal()`.

I marked as optional because the gain here is limited, but if you find yourself regularly wrapping the call in `require.NoError()` then it cleans all of those up.

```go
func VerifyInitializedStruct(tb testing.TB, s interface{}) {
    tb.Helper()
```

Reply:
I totally blanked on this pattern when putting together the PR. Thanks for the reminder! Updated